eng/common/Update-Change-Log.ps1

```diff
@@ -12,7 +12,8 @@ param (
   [Parameter(Mandatory = $true)]
   [String]$ChangeLogPath,
   [String]$Unreleased = $True,
```

Comment:
Why are you declaring booleans as strings? PowerShell way is `[switch] $Unreleased`. Specifying it means `$true`, but you can pass `:$false` or `:$true` explicitly like `-Unreleased:$false`.

Reply:
Don't know :)

Reply:
I'll leave it as is for now.

Reply:
Yes lets not change too much here currently. @chidozieononiwu does have issue https://github.com/Azure/azure-sdk-tools/issues/829 which we will use as a clean-up effort for our various PS scripts.